jwst/lib/pointing_summary.py

```diff
@@ -24,13 +24,13 @@
 >>> print(t.columns)
     <TableColumns names=('exposure','target','v1','refpoint','delta_v1','delta_refpoint')>
 >>> print(t["delta_v1"])
-          delta_v1
```

Comment:
Did I remove too many whitespaces here? The following 2 lines needed to have 4 spaces removed, while this one was 6, in order to get it to line up visually the same as the others. How does the spacing make a difference to the tests? The removal of 4 spaces in lines 31-33 seems to have worked for all test instances.

Reply:
One can round in doctests by simply printing a truncated version of the results. For this particular example use the following. Using old-style printing to cut down on quotation mark issues.

```
>>> "%.5f" % t["delta_v1"]

Reply:
And of course, fix the comparison to match the precision.

Reply:
and maybe more decimal places? looks like no more than 13.

Reply:
> And of course, fix the comparison to match the precision.

Sorry for being dim, but fix which comparison to match what precision? And where are more decimal places needed? It seems to me that part of the problem we already have is that too many decimal places are being used for the comparisons and we need to cut them back.

Reply:
Hmmm, I see the issue. because it is printing a column. Will have to play but almost assuredly there is a way to limit the precision of the printing of table columns. If not another way is to assign the value then print it. As follows:

```
>>> delta_v1 = t["delta_v1"][0]
>>> %.13f % delta_v1
0.137127271640